interop-testing/src/main/java/io/grpc/testing/integration/TestServiceImpl.java

```diff
@@ -447,6 +447,7 @@ private StreamingOutputCallResponse toResponse() {
   /**
    * Creates a buffer with data read from a file.
    */
+  @SuppressWarnings("Finally") // Not concerned about suppression; expected to be exceedingly rare
   private ByteString createBufferFromFile(String fileClassPath) {
```

Comment:
Are we allowed java7 in test code?  That might let us use try-with safely and avoid the warning suppression